selection/algorithms/epic_algorithm.py

```diff
@@ -49,11 +51,11 @@ def _calculate_best_indexes(self, workload):
                     self._evaluate_combination(index_combination + [candidate],
                                                best, current_cost)
 
+            for candidate in multi_attribute_index_candidates:
```

Comment:
`candidate` is not a multi_attribute_index candidate

Reply:
We could refactor _attach_to_indexes() to become a generate_multi_column_candidates() to make the code clearer.

Reply:
I believe this could be improved by renaming. For example `candidate` to `attribute` and `multi_attribute_index_candidates` to `extension_attribute_candidates`?